src/software/simulated_tests/example_play_test.cpp

```diff
@@ -14,6 +14,7 @@ class ExamplePlayTest : public SimulatedTestFixture
 
 TEST_F(ExamplePlayTest, test_example_play)
 {
+    enableVisualizer();
```

Comment:
this should be removed for the final submission (CI won't pass)

Reply:
removed

Reply:
removed